### PR TITLE
Ws 001 sightings repo

### PR DESCRIPTION
**Pull request by doctone**

### CODE REVIEW

File: backend/Models/Request/CreateSightingRequest.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{
+    public class CreateSightingRequest
+    {
+
+        [Required]
+        public DateTime Date { get; set; }
+
+        [Required]
+        [StringLength(70)]
+        public string LocationName { get; set; }
```
Comment (TimLeach635) [inline backend/Models/Request/CreateSightingRequest.cs:15]: I notice that, as written, every single sighting will generate a new, distinct location, even if somebody puts in the same name! This is fine for an MVP, but I want to make sure you are aware of the limitations of this choice, and confirm that you made this choice consciously

File: backend/Repositories/SightingsRepo.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    public class SightingsRepo
+    {
+        private readonly WhaleSpottingDbContext _context;
+
+        public SightingsRepo(WhaleSpottingDbContext context)
+        {
+            _context = context;
+        }
+        public List<Sighting> GetAllSigthings()
```
Comment (TimLeach635) [inline backend/Repositories/SightingsRepo.cs:31]: Typo!

File: backend/Repositories/SightingsRepo.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        public void Delete(int id)
+        {
+            var sighting = GetById(id);
+            _context.Sightings.Remove(sighting);
+            _context.SaveChanges();
+        }
+    }
+
+
+}
```
Comment (TimLeach635) [inline backend/Repositories/SightingsRepo.cs:71]: Don't have to change this now, but in general it's preferred to have exactly one blank line at the end of every file (you may have noticed in demos I compulsively do this when I create a new file)

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
An important change to make here!

File: backend/Models/Request/CreateSightingRequest.cs (hunk 1 above)
Comment (doctone) [reply backend/Models/Request/CreateSightingRequest.cs:15]: thanks! We had planned to have the location input as a drop down on the frontend (as per planning on mon) - will we have to change it here in order to do this?

**Review by TimLeach635 — APPROVED** [review]

**Pull request merged**